packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -355,10 +356,39 @@ export class StripeWebhookHandler extends StripeHandler {
     request: AuthRequest,
     event: Stripe.Event
   ) {
-    const sub = event.data.object as Stripe.Subscription;
-    let uid;
     try {
-      ({ uid } = await this.sendSubscriptionDeletedEmail(sub));
+      const sub = event.data.object as Stripe.Subscription;
+      const customer = await this.stripeHelper.expandResource(
+        sub.customer,
+        CUSTOMER_RESOURCE
+      );
```

Comment:
I see. You're expanding the `customer` here to get the `uid` from the customer metadata instead of from calling `sendSubscriptionDeletedEmail`, which we don't want to do in this case.